.github/CODEOWNERS

```diff
@@ -341,11 +340,10 @@ yarn.lock                                                @getsentry/owners-js-de
 /src/sentry/tasks/symbolication.py                                   @getsentry/owners-native
 /src/sentry/utils/appleconnect/                                      @getsentry/owners-native
 /tests/sentry/tasks/test_low_priority_symbolication.py               @getsentry/owners-native
-
-# Shared ownership of reprocessing as transitionary phase
-/src/sentry/tasks/reprocessing.py                                    @getsentry/owners-ingest @getsentry/owners-native
-/src/sentry/tasks/reprocessing2.py                                   @getsentry/owners-ingest @getsentry/owners-native
-/src/sentry/reprocessing2.py                                         @getsentry/owners-ingest @getsentry/owners-native
+/src/sentry/tasks/reprocessing.py                                    @getsentry/owners-native
+/src/sentry/tasks/reprocessing2.py                                   @getsentry/owners-native
+/src/sentry/reprocessing2.py                                         @getsentry/owners-native
```

Comment:
😱 🙀 
![image](https://user-images.githubusercontent.com/580492/230408364-4ad2a06a-7f75-4892-afa4-ce7e6a45dc96.png)

Reply:
Let me know if I went too far here. I was just making assumptions based on @ashwoods email.

Reply:
![](https://media.tenor.com/wbh2az7l64AAAAAC/modern-family-pray.gif)

Reply:
We now have @getsentry/team-processing @getsentry/owners-processing. Re-processing is still an open topic, but I think we should own it instead of leaving it unowned, and expectations adjusted ;)